### PR TITLE
Updates to building connections application flow

### PR DESCRIPTION
**Pull request by davidrapson**
- Update questions, add new project theme and address fields
- Update e2e tests. Include new fields and directly inject textarea text to speed up tests
- Add fieldset introduction and option choices to startpage

### CODE REVIEW

**Comment by davidrapson** [comment]
@mattandrews This is ready to review now

File: controllers/apply/building-connections/fields.js [hunk 1]
```diff
@@ -33,11 +33,14 @@ const currentWork = [
                 type: 'textarea',
                 lengthHint: LENGTH_HINTS.MANY_PARAS,
                 isRequired: true,
-                label: 'How is your current work committed to helping to prevent or reduce loneliness?',
+                label: 'How is your current work committed to helping to prevent or reduce lonelines?',
```
Comment (mattandrews) [inline controllers/apply/building-connections/fields.js]: Typo got introduced here!

File: controllers/apply/building-connections/fields.js [hunk 2]
```diff
@@ -33,11 +33,14 @@ const currentWork = [
...
                 helpText: `<ul>
-                    <li>Describe why you are working on loneliness</li>
-                    <li>Describe what impact your work has made on tackling loneliness so far, e.g. evidence of the difference it is making and the impact on beneficiaries</li>
-                    <li>Tell us about the effectiveness of your approach and skills of your staff</li>
+                    <li>Describe the work you are currently delivering and how it helps prevent or reduce loneliness</li>
+                    <li>Describe what impact your work has made on tackling loneliness so far,
+                        e.g. evidence of the difference it is making, the impact on people and
+                        communities and how many people are currently benefiting</li>
+                    <li>Tell us about the effectiveness of your approach and your
+                        organisation’s experience and expertise in of working with loneliness.</li>
```
Comment (mattandrews) [inline controllers/apply/building-connections/fields.js]: "expertise in of working" = typo?

File: controllers/apply/building-connections/fields.js [hunk 3]
```diff
@@ -204,6 +272,49 @@ const organisationDetails = [
             }
         ]
     },
+    {
+        legend: 'Registered organisation address',
+        introduction: `If your application is succesful we will use this address to send out a conditional offer letter.`,
```
Comment (mattandrews) [inline controllers/apply/building-connections/fields.js]: Typo on "succesful" (missing an S)

**Comment by davidrapson** [comment]
Serves me right for blind copy-pasting copy.

**Pull request merged**